N/neofetch/build_tarballs.jl

```diff
@@ -34,4 +34,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat = "1.6")
```

Comment:
Again, this is a platform-independent package, restricting the julia compat does nothing useful.

Reply:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```